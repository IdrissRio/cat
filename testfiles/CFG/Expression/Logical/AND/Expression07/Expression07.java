public class Expression07 {
  void m(int a, int b) {
    if (a < 10 && b > 10) {
      int c;
    }
    int d;
  }
}