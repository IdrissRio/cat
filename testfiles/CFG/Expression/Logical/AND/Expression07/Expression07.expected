NodeName: Entry[15] Successors: [Exit[16]]
NodeName: Exit[16] Successors: []
NodeName: Entry[7] Successors: [Exit[8]]
NodeName: Exit[8] Successors: []
NodeName: Entry[29] Successors: [VarAccess[49]]
NodeName: VarAccess[49] Successors: [IntegerLiteral[50]]
NodeName: IntegerLiteral[50] Successors: [LTExpr[48]]
NodeName: LTExpr[48] Successors: [VarAccess[52], VariableDeclarator[70]]
NodeName: VarAccess[52] Successors: [IntegerLiteral[53]]
NodeName: IntegerLiteral[53] Successors: [GTExpr[51]]
NodeName: GTExpr[51] Successors: [VariableDeclarator[61], VariableDeclarator[70]]
NodeName: VariableDeclarator[61] Successors: [VariableDeclarator[70]]
NodeName: VariableDeclarator[70] Successors: [Exit[44]]
NodeName: Exit[44] Successors: []

NodeName: Exit[16] Predecessor: [Entry[15]]
NodeName: Entry[15] Predecessor: []
NodeName: Exit[8] Predecessor: [Entry[7]]
NodeName: Entry[7] Predecessor: []
NodeName: Exit[44] Predecessor: [VariableDeclarator[70]]
NodeName: VariableDeclarator[70] Predecessor: [LTExpr[48], GTExpr[51], VariableDeclarator[61]]
NodeName: LTExpr[48] Predecessor: [IntegerLiteral[50]]
NodeName: IntegerLiteral[50] Predecessor: [VarAccess[49]]
NodeName: VarAccess[49] Predecessor: [Entry[29]]
NodeName: Entry[29] Predecessor: []
NodeName: GTExpr[51] Predecessor: [IntegerLiteral[53]]
NodeName: IntegerLiteral[53] Predecessor: [VarAccess[52]]
NodeName: VarAccess[52] Predecessor: [LTExpr[48]]
NodeName: VariableDeclarator[61] Predecessor: [GTExpr[51]]
